src/Http/Http/src/SendFileFallback.cs

```diff
@@ -36,19 +63,14 @@ public static async Task SendFileAsync(Stream destination, string filePath, long
 
             int bufferSize = 1024 * 16;
```

Comment:
I think we want the buffer size to be 1 actually. At least for the pipe case. No point of having the FileStream keep an internal buffer here.

Reply:
👍 Fortunately, `FileStream.ReadAsync(destination)` doesn't allocate an internal buffer for reads unless `destination.Length < bufferSize` which it should never be if it's 1.

Reply:
@davidfowl Do you mean 1 as in `int bufferSize = 1024;` or `int bufferSize = 1;`?

Reply:
bufferSize = 1. We want to avoid the FileStream's internal buffer completely for this operation.